src/api/app/views/webui/comment/_comment_field.html.haml

```diff
@@ -28,7 +28,7 @@ data: { preview_message_url: preview_comments_path, message_body_param: 'comment
         placeholder: 'Write your comment here... (Markdown markup is supported)', required: true,
         class: 'w-100 form-control comment-field message-field'
       .tab-pane.fade{ id: "preview_#{element_suffix}", role: 'tabpanel', 'aria-labelledby': 'preview-message-tab' }
-        .comment-preview.message-preview.border-bottom.border-gray-300.my-3
+        .comment-preview.message-preview.my-3
```

Comment:
We could remove the definition of `border-gray-300` in `src/api/app/assets/stylesheets/webui/borders.scss`. We no longer use this CSS class, I guess.

Reply:
Good catch! Dropped